### ISSUE TITLE
TMP breaks New Window with Tridactyl

### ISSUE THREAD

**Issue by Massimo-B** [onemen/TabMixPlus#123]
Cross-referenced ticket: https://github.com/tridactyl/tridactyl/issues/4146

Using CTRL+N to open a new window by Firefox, the window opens, but is getting closed right after appearing.
I tried all addon combinations, reduced to the minimum set of addons:

- boostrap loader
- Tab Mix Plus
- Tridactyl: Beta

Having _Tridactyl_ alone or _Tab Mix Plus_ alone works, but the combination breaks CTRL+N.

As Newtab setting in Tab Mix Plus I followed
https://github.com/tridactyl/tridactyl/issues/4145#issuecomment-1070834502
and have left `New Tab Page, Address` in Tab Mix Plus empty.
:newtab and :winopen are working fine.

**Comment by onemen** [comment]
can you attach a file with your Tab Mix preferences ?

**Comment by onemen** [comment]
I can confirm this issue.

There is a bug in Tab mix function that `Prevent blank tabs when downloading files`, it can trigger close tab for tab opens by Tridactyl.

As a workaround until i will upload new version set `extensions.tabmix.enablefiletype` to `false` in `about:config`

**Comment by bovine3dom** [comment]
In Tridactyl, `:winopen` roughly does

```
browser.windows.create({url: "https://this-is-a-dummy-url-to-fix-a-firefox-bug.invalid"}).then(win => browser.tabs.update(win.tabs[0].id, {loadReplace: true}, url: "URL GOES HERE"))
```

https://github.com/tridactyl/tridactyl/blob/a72b2b4a3e7c4067859943a0ee14d6d35c16e209/src/excmds.ts#L2965

(it does this to make sure that focus ends up on the page and not the URL bar)

**Comment by Massimo-B** [comment]
> As a workaround until i will upload new version set `extensions.tabmix.enablefiletype` to `false` in `about:config`

That works. Thank you very much!
Which other effect will that have?

Please close this ticket when releasing the new version.

**Comment by onemen** [comment]
> Which other effect will that have?

Setting `extensions.tabmix.enablefiletype` to `false`  disable `Prevent blank tabs when downloading files`

**Comment by Massimo-B** [comment]
Hi. Today I noticed that FF did update the Tab Mix Plus, but Version is still unchanged v1.0.0-pre.6, just like the last github release from (2022-03-12). Are you able to submit changes without touching the release tag?

**Comment by onemen** [comment]
The last update was on 2022-03-12 few hours after i've uploaded version v1.0.0-pre.6, it was just minor change that did not requiter a version change (c00c954a0326d8c7c527c7267003a95810132ffd, 72b2b1d9cda5037da65ea7417e9c46155c457e1e) .

The fix for this bug will be in version v1.0.0-pre.7
